src/monitord/main.c

```diff
@@ -162,6 +162,13 @@ int main(int argc, char **argv)
         merror_exit(CONFIG_ERROR, cfg);
     }
 
+    /* Setting default values and reading global configuration */
+    Config.agents_disconnection_time = 20;
```

Comment:
We should create a method to set default values and read the configuration, as in Remoted.

Reply:
Now, the _Config global_ field is read at the same time that **mond**, but a new method was not created.